primitives/message-lane/src/lib.rs

```diff
@@ -79,7 +79,10 @@ pub struct InboundLaneData<RelayerId> {
 	/// 2) all incoming messages are rejected if `proof-of(outbound-lane.state).latest_received_nonce` is
 	///    equal to `this.latest_confirmed_nonce`.
 	/// Given what is said above, all nonces in this queue are in range (latest_confirmed_nonce; latest_received_nonce].
-	pub relayers: VecDeque<(MessageNonce, RelayerId)>,
+	/// Also, when a relayer sends a single message, both of MessageNonces are the same.
+	/// When relayer sends messages in a batch, the first arg is the lowest nonce, second arg the highest nonce.
+	/// Multiple dispatches from the same relayer one are allowed.
+	pub relayers: VecDeque<(MessageNonce, MessageNonce, RelayerId)>,
```

Comment:
We should probably think of a new limit now per max messages per batch or reuse `MaxUncofirmedMessagesAtInboundeLane` to defacto set overall msgs capacity at `MaxUncofirmedMessagesAtInboundeLane * MaxUncofirmedMessagesAtInboundeLane`

Reply:
max number of messages in batch is naturally limited by weight, I don't think we need an extra limit. `MaxUnconfirmedMessagesAtInboundLane` is there to make sure we don't end up having too many items in this collection.

I wonder what happens now if the collection is full (i.e. `.len() == MaxUnconfirmed..`) - is the last relayer still allowed to submit more messages? If yes then it's a potential griefing attack for the relayers. We should disallow submitting any more, even if it doesn't grow the collection - can we add a test for this?